pkg/errs/errors.go

```diff
@@ -0,0 +1,47 @@
+package errs
```

Comment:
The name of the package doesn't really fit imo, since this isn't a generic errors package (to contrast see the standard package).
IIRC these wrappers are specifically used for debug errors right? So I think `debug` or `debugerr` would be a more fitting name.

Reply:
Yep, it is not debug as well. Lets call it errfmt or something like it. Not a big deal IMO.